### DISCUSSION TITLE
Metro-DR (Synchronous replication across 2 physical servers)

### DISCUSSION THREAD

**Discussion by Aniket-IN** [openebs/openebs#3698] [General]
Hi, 

I'm super new to the whole DevOps world. 
Have worked with Docker a lot, started learning K8s. 

I have two physical servers few miles away (latency is 3-4ms) 
Looking for a Zero RPO solution, like we should always have the Primary Server replicated to the Standy Server.
We should be easily able to failover to `Standby` server and make that primary when needed.

And same way, switch back to `Primary` server safely after syncing it back.

Can someone please, give some guidance what things I should learn? 
There are many things, and I'm totally confused from where to start.

Thank you so much in advance :)

**Comment by niladrih** [comment]
Synchronous replication for storage...? Mayastor does that. Not sure if you're asking about storage though.
